rpc/client/http/http.go

```diff
@@ -268,6 +272,15 @@ func (c *baseRPCClient) broadcastTX(route string, tx types.Tx) (*ctypes.ResultBr
 	return result, nil
 }
 
+func (c *baseRPCClient) broadcastTXs(route string, txs types.Txs) (*ctypes.ResultBroadcastTx, error) {
+	result := new(ctypes.ResultBroadcastTx)
```

Comment:
Is it possible for this to return []*ctypes.ResultBroadcastTx so that you can capture the response from each transaction?